### ISSUE TITLE
Version 1.4 Italian Translation

### ISSUE THREAD

**Issue by CoralineAda** [ContributorCovenant/contributor_covenant#236]
Version 1.4 needs an Italian translation. See the diff at https://github.com/ContributorCovenant/contributor_covenant/blob/master/diffs/1_3_vs_1_4.patch. 

Attn @jodosha

**Comment by Dasc3er** [comment]
If no one has already done it, I would be happy to contribute: [https://github.com/Dasc3er/contributor_covenant](https://github.com/Dasc3er/contributor_covenant)

This is my fork with the Italian translation, but I believe itshould be checked by at least another Italian speaker.
Should I also create a pull request?

**Comment by CoralineAda** [comment]
Yes, please open a pull request and then I will help find another Italian speaker to review. Thanks!

**Comment by AmberBee** [comment]
Hi, I'm Italo-American, native Italian language but very good in English too ;)  Can I help?

**Comment by Dasc3er** [comment]
@AmberBee If you want, I created a Pull Request (https://github.com/ContributorCovenant/contributor_covenant/issues/346) with the Italian translation based on 1.4.0...
